tests/avro/test_cached_client.py

```diff
@@ -140,19 +142,25 @@ def hash_func(self):
 
     def test_cert_no_key(self):
         with self.assertRaises(ValueError):
-            self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                     cert_location='/path/to/cert')
+            self.client = CachedSchemaRegistryClient({
```

Comment:
Why are all these changed to the configdict instead of their kwargs?

Reply:
> Why are all these changed to the configdict instead of their kwargs?

The constructor for CachedSchemaRegistryClient is changed: https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/avro/cached_schema_registry_client.py#L54, a configdict is needed here